### ISSUE TITLE
Visitor analytics

### ISSUE THREAD

**Issue by nsiregar** [nsiregar/pegelinux#55]
Should we added visitor analytic on pegelinux.id?

/polls "Google Analytics" "Matomo/Piwik" "No Analytics"

**Comment by samsunoor** [comment]
google analytics seems a bad idea. most adblocker blacklists that script.

**Comment by nsiregar** [comment]
harusnya ada opsi untuk matomo, tapi ga tau ga nongol

**Comment by samsunoor** [comment]
matomo atau piwik lebih bagus deh

**Comment by wildtype** [comment]
Well if GA is blocked, so be it. The visitor choose not to be tracked, lets respect their decision. I think all popular analytics tools is blocked, so it'll be the same.

**Comment by samsunoor** [comment]
TIL
So be it, then...

**Comment by wildtype** [comment]
btw, matomo ada hosted yang hratis kah?

**Comment by nsiregar** [comment]
ga ada, makanya ane prefer ke GA

**Comment by wildtype** [comment]
Ngehost sendiri nambah ongkos sih ya. Tapi mungkin ga perlu resources gede bgt kalo dipake cuma di 1 situs dan masih sepi.

**Comment by nsiregar** [comment]
harusnya bisa sih satu paket ama yang sekarang.

**Comment by wildtype** [comment]
Jangan disatuin aja sih mesin & dbnya, analytics write datanya gila-gilaan, kalo tepar, ntar situs utamanya bisa ikutan kena.

**Comment by nsiregar** [comment]
nah, makanya ane prefer ke GA. udah ga perlu mikirin ketika kejadian tersebut terjadi

**Comment by samsunoor** [comment]
Mending google analytics aja deh.
Etapi pentingnya apa dulu nih ?
UI UX ? performance ? A/B test ?

**Comment by wildtype** [comment]
Kumpulin data dulu, kepentingannya belakangan, wkwkwk :rofl:
